goetas/twital-bundle/1.3/config/packages/twital.yaml

```diff
@@ -0,0 +1,7 @@
+goetas_twital:
+    full_twig_compatibility: true # see http://twitalbundle.readthedocs.io/en/latest/#full-twig-compatibility
+
+## add here your custom extension handlers, by default Twital parses *.html.twital, *.xml.twital and *.xhtml.twital files
```

Comment:
@Nyholm If you think is a better idea is can completely remove this comment and let the users find out this detail via documentation.

Was thinking to put just a note "By default Twital parses *.html.twital, *.xml.twital and *.xhtml.twital files, more info about it on http://twitalbundle.readthedocs.io/en/latest/#source-adapters"